src/pages/FriendsList.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-console */
+import ListHeader from '../components/list/ListHeader';
+import { useRecoilValue } from 'recoil';
+import { userState } from '../store/recoil';
+import styled from 'styled-components';
+import Squircle from '../components/user/Squircle';
+import { ListBody } from '../styles/Container';
+import UserListItem from '../components/list/UserListItem';
+import { useState } from 'react';
+import SearchBox from '../components/list/SearchBox';
+
+const FriendsList = () => {
+    const userList = useRecoilValue(userState);
+    const me = userList[0];
+    const friends = userList.slice(1);
+    const [search, setSearch] = useState(false);
```

Comment:
```suggestion
    const [isOpend, setIsOpend] = useState(false);
```
flag 변수명이 직관적이지 않은것 같습니다. isOpened 정도가 적당한것 같습니다.